docs/guides/cache/azblob.md

```diff
@@ -1,62 +1,46 @@
 # Azure Blob Storage cache storage
 
-> **Warning**
->
-> The `azblob` cache is currently unreleased. You can use it today, by using
-> the `moby/buildkit:master` image in your Buildx driver.
+> This cache backend is unreleased. You can use it today, by using the
+> `moby/buildkit:master` image in your Buildx driver. {: .warning }
```

Comment:
Ah I did this so that we'd get the neat little view in GitHub for users who browse the docs through that:

![image](https://user-images.githubusercontent.com/7352848/191527782-5680a874-6daa-4097-9f29-bcb3ba756277.png)

Not quite sure if there's a way of getting it to show in GitHub and also docs.docker.com? @crazy-max :eyes:

Reply:
`> **Note**` works on docs.docker.com too

Reply:
See 
* https://github.com/docker/buildx/blob/master/docs/guides/bake/compose-file.md#env-file
* https://docs.docker.com/build/customize/bake/compose-file/#env-file

Reply:
Ah, sorry, I didn't realize. Will add it back ofc!

Reply:
I still see the ` {: .warning }` style in a few places, oops :eyes:

Reply:
oh I didn't realize we should take out the liquid tags. Will do.